css/style.css

```diff
@@ -408,6 +408,10 @@ a:hover {
   display: none;
 }
 
+.form-section form label {
+  display: none;
+}
```

Comment:
## adding labels (CSS change)
- Add display none to the labels so that the portfolio can match the template